system_tests/logging_.py

```diff
@@ -50,8 +51,12 @@ def setUp(self):
         self.to_delete = []
 
     def tearDown(self):
+        from gcloud.exceptions import NotFound
         for doomed in self.to_delete:
-            doomed.delete()
+            try:
+                doomed.delete()
+            except NotFound as error:
+                print('Unable to delete resource: %s' % error)
```

Comment:
This is due to #1657 (the loggers "created" via the `/entries:write` API cannot be deleted).